lgc/interface/lgc/Pipeline.h

```diff
@@ -133,8 +133,8 @@ struct Options {
   bool fullSubgroups;                  // Use full subgroup lanes
   unsigned nggVertsPerSubgroup;        // How to determine NGG verts per subgroup
   unsigned nggPrimsPerSubgroup;        // How to determine NGG prims per subgroup
-  unsigned shadowDescriptorTable;      // High dword of shadow descriptor table address, or
-                                       //   ShadowDescriptorTableDisable to disable shadow descriptor tables
+  unsigned shadowDescriptorTable;      // High dword of shadow descriptor table address
+  bool enableFmask;                    // Enable Fmask
```

Comment:
A more descriptive comment may be helpful:
```suggestion
  bool enableFmask;                    // Whether to use Fmasks when loading from MSAA images
```